noarkexchange/src/main/java/no/difi/meldingsutveksling/noarkexchange/PayloadUtil.java

```diff
@@ -103,4 +105,26 @@ public static String queryPayload(PutMessageRequestWrapper message, String xpath
         return result;
     }
 
+    public static Object unmarshallPayload(Object payload) throws JAXBException {
+        String p;
+        Object msg = null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "msg". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1854)